cmd/notary/integration_test.go

```diff
@@ -81,6 +81,43 @@ func setupServer() *httptest.Server {
 	return httptest.NewServer(setupServerHandler(storage.NewMemStorage()))
 }
 
+// Initializes a repo with existing key
+func TestInitWithMultipleRootKeys(t *testing.T) {
+	// -- setup --
+	setUp(t)
+
+	tempDir := tempDirWithConfig(t, "{}")
+	defer os.RemoveAll(tempDir)
+
+	server := setupServer()
+	defer server.Close()
+
+	tempFile, err := ioutil.TempFile("", "targetfile")
+	require.NoError(t, err)
+	tempFile.Close()
+	defer os.Remove(tempFile.Name())
+
+	// -- tests --
+
+	// generate root key produces a single root key and no other keys
+	_, err = runCommand(t, tempDir, "key", "generate", data.ECDSAKey)
+	require.NoError(t, err)
+	rootIDs, _ := assertNumKeys(t, tempDir, 1, 0, true)
+	var rootFiles []string
+	for _, rID := range rootIDs {
+		rootFiles = append(rootFiles, filepath.Join(
+			tempDir, "private", "root_keys", rID+".key"))
+	}
+
+	// init repo
+	_, err = runCommand(t, tempDir, "-s", server.URL, "init", "gun", "--rootkey", rootFiles[0])
+	require.NoError(t, err)
+
+	// publish repo
+	_, err = runCommand(t, tempDir, "-s", server.URL, "publish", "gun")
+	require.NoError(t, err)
```

Comment:
Apologies for the trouble - would it also be possible to add some or all of the following negative test cases?  Similar test cases may already exist for importing a root key.

1.  Passing a non-existent file with the `--rootkey` flag should cause the command to fail
1.  Passing a non-PEM file with the `--rootkey` flag should cause the command to fail
1.  Passing an unencrypted PEM file with the `--rootkey` flag should cause the command to fail
1.  Failure to enter the correct passphrase with a valid `--rootkey` file should cause the command to fail
1.  Passing a targets or snapshot key with the `--rootkey` flag should cause the command to fail, even if the correct password is entered